dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -212,14 +220,27 @@ async def _get_job_activity_info(self, timeout: int) -> RayActivityResponse:
             )
 
             num_active_drivers = 0
+            latest_job_end_time = 0
             for job_table_entry in reply.job_info_list:
                 is_dead = bool(job_table_entry.is_dead)
                 in_internal_namespace = job_table_entry.config.ray_namespace.startswith(
                     "_ray_internal_"
                 )
+                latest_job_end_time = max(latest_job_end_time, job_table_entry.end_time)
```

Comment:
Are end times always numbers? I ask becaue `max(2, None)` is an exception.

Reply:
I believe the `end_time` should be 0 if a job hasn't started yet, but not sure if the schema of this response enforces it so it's probably a good idea to check here